src/core/lib/iomgr/event_engine/iomgr.h

```diff
@@ -17,8 +17,16 @@
 
 #include <grpc/event_engine/event_engine.h>
 
-// This can be called anywhere in the EE-based iomgr impl where we need to
-// access the global EE instance.
+/// This can be called anywhere in the EventEngine-based iomgr impl where we
+/// need to access the global EventEngine instance.
```

Comment:
Why 3 slashes?

Reply:
doxygen formatting https://www.doxygen.nl/manual/docblocks.html. I'm actually not sure when if ever we generate docs for code in `src/core`, but if ever someone does, triple slash will make these comments show up there.

Reply:
Cool. TIL.